### DISCUSSION TITLE
Running flexget on another machine

### DISCUSSION THREAD

**Discussion by epross672** [Flexget/Flexget#3937] [Q&A]
My main flexget machine is old and has python 3.6.9 and it is not possible to upgrade python on that box.

I have too much data to move to my new machine that has the latest python version and flexget version.  Is there a way to have flextget on the new machine download and move sort etc files on the old box?

**Comment by gazpachoking** [comment]
You could use network mounts to mount the drives on your new machine. It's probably also possible to install newer python or Flexget on your other machine even if there aren't native packages for your OS.
